development-guide.md

```diff
@@ -0,0 +1,112 @@
+# Promitor Development Guide
+
+This guide explains the layout of the Promitor repository, along with the tools
+and technologies required to develop Promitor.
+
+## Repository Structure
+
+The promitor repository is made up of a number of different directories:
+
+- `/` - contains documentation about the repository including the main README file
+  and some configuration files for various tools used for development.
+- `/.github` - contains github specific files including issue and PR templates.
+- `/.vscode` - contains shared configuration files for [VS Code](https://code.visualstudio.com/).
+- `/build` - contains the configuration files for Promitor's CI process.
+- `/charts` - contains Promitor's [Helm](https://helm.sh/) chart.
+- `/deploy` - TODO.
```

Comment:
I can help you with that:

`contains the automation that is being used to manage Promitor, such as the automated updates concerning new Docker images in a pull request.`